src/hooks/useDefaultHeader.ts

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: MPL-2.0
```

Comment:
Geen test? Ik zie geen uitsluiten van tests. Gaat hiermee ons coveragepercentage niet onnodig mee naar beneden?

Reply:
deze functies worden al ergens gebruikt in een test.